src/Tools/IdeCoreBenchmarks/FindReferencesBenchmarks.cs

```diff
@@ -5,16 +5,20 @@
 #nullable disable
 
 using System;
```

Comment:
benchmark code is effectively a way for me to quickly test things liek FindRefs and NavTo on different DBs, comparing the results.  they're not really productino code, and more a scratch pad to answer questions quickly on a real world codebase.